docs/TROUBLESHOOTING.md

```
@@ -303,9 +698,72 @@ cortex rollback <installation-id>
+cat ~/.cortex/logs/cortex.log
+```
+
+### Report a bug
+
+1. Gather information:
+   ```bash
+   cortex status > cortex-status.txt
+   cat ~/.cortex/logs/cortex.log >> cortex-status.txt
+   ```
+
+2. Open an issue: https://github.com/cortexlinux/cortex/issues
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)